<?xml version="1.0" ?>
<launch>
  <param name="/local_planner/replan_period" value="0.05"/>
  <param name="/planner/range_next_waypoint" value="0.3"/>
  <param name="/planner/low_reso" value="0.2"/>
  <param name="/head_x_offset" value="0.1"/>
  <param name="/hop_range" value="0.05"/>
  <node pkg="local_planner_hop" type="decision_maker.py" name="decision_maker" output="screen"/>
  <node pkg="local_planner_hop" type="cheetah_planner.py" name="cheetah_planner" output="screen"/>
  <node pkg="local_planner_hop" type="control.py" name="control" output="screen"/>

</launch>
